service/src/main/java/greencity/service/EmailServiceImpl.java

```diff
@@ -143,11 +143,12 @@ public void sendCreatedNewsForAuthor(EcoNewsForSendEmailDto newDto) {
      * @author Volodymyr Turko
      */
     @Override
-    public void sendVerificationEmail(Long id, String name, String email, String token, String language) {
+    public void sendVerificationEmail(Long id, String name, String email, String token, String language,
+        boolean isUbs) {
         Map<String, Object> model = new HashMap<>();
         model.put(EmailConstants.CLIENT_LINK, clientLink);
         model.put(EmailConstants.USER_NAME, name);
-        model.put(EmailConstants.VERIFY_ADDRESS, clientLink + "#/?token="
+        model.put(EmailConstants.VERIFY_ADDRESS, clientLink + "#/" + (isUbs ? "ubs" : "") + "?token="
```

Comment:
Also, it would be best if you implemented this logic in JSP which is located [here](https://github.com/ita-social-projects/GreenCityUser/blob/master/core/src/main/resources/templates/email/verify-email-page.html)